site/getting-started/overview.md

```diff
@@ -18,79 +18,80 @@ head:
       content: Learn what Account Kit is, how it works, and how you can use it to integrate smart accounts in your app.
 ---
 
-# Why Account Kit?
+# Why Embedded Accounts?
```

Comment:
it's a bit confusing for the Account Kit docs' overview to have the first line be about Embedded Accounts with no mention of how its powered by Account Kit.